department-of-veterans-affairs/caseflow
# Calvin/appeals 25907 fixes

Resolves [APPEALS-25907](https://jira.devops.va.gov/browse/JIRA-25907)

# Description
Fixed issue where using the rake generator multiple times would grab all previously generated legacy appeals and assign to user. This was happening due to the lack of changes being made to the STAFF attributes when it was a hearing or distribution task

## Acceptance Criteria

## Testing Plan

1. Run rake db:generate_legacy_appeal[false,true] multiple times on different users in your terminal
2. Check their queue and ensure that all generated legacy appeals are not pulled in to the specified user
